libs/utils/jest.config.js

```diff
@@ -7,4 +7,14 @@ module.exports = {
   ...shared,
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts'],
+  coverageThreshold: {
+    ...shared.coverageThreshold,
+    // Moved from frontends/election-manager with pre-existing low coverage:
+    ['src/votecounting.ts']: {
```

Comment:
Unnecessary computed property.
```suggestion
    'src/votecounting.ts': {
```